var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -13,10 +13,10 @@ class Xrootd(CMakePackage):
 
     homepage = "https://xrootd.slac.stanford.edu"
```

Comment:
Do you want to also change the homepage to https://xrootd.web.cern.ch?

Reply:
Right, we should change this one too.
```suggestion
    homepage = "https://xrootd.web.cern.ch"
```